gradle/javadoc.gradle

```diff
@@ -33,26 +33,25 @@ task docs(type: Javadoc) {
 
 def utilProject = project(':util')
 allprojects {
-    afterEvaluate { project ->
```

Comment:
The reason I removed `afterEvaluate` is due to this warning:
```
Using method Project#afterEvaluate(Closure) when the project is already evaluated has been deprecated. This will fail with an error in Gradle 7.0. The configuration given is ignored because the project has already been evaluated. To apply this configuration, remove afterEvaluate.
```

Reply:
I will address this in JAVA-3574.